### ISSUE TITLE
not able to trigger broker app (Microsoft authenticator app)

### ISSUE THREAD

**Issue by NikhilRathore0** [AzureAD/microsoft-authentication-library-for-objc#2127]
Hello everyone need help regarding MSAL authentication using the broker app i have added the required package and the code change to my project but still I am not able to trigger the Microsoft authenticator app using MSAL for authentication

**Comment by oldalton** [comment]
Make sure you closely follow steps outlined in this article: https://learn.microsoft.com/en-us/entra/msal/objc/install-and-configure-msal#ios-only-steps

**Comment by NikhilRathore0** [comment]
Hello @oldalton thanks for the suggestion. I followed all the points recommended in this doc but it is still not working can you please help me with this I need to force MSAL to authenticate with the broker(Microsoft authenticator) app

**Comment by NikhilRathore0** [comment]
Hello @oldalton is there any flutter implementation for the MSAL authentication

**Comment by NikhilRathore0** [comment]
Hi @oldalton, thank you for your response. In one of our client projects, we implemented MSAL for authentication. However, we encountered an issue where the iOS/iPad operating system uses Safari browser as its default, which is blocked by the client's Azure policy. The client has requested that we change to using Edge as the default browser for the MSAL web view, but unfortunately, the iOS/iPad operating system does not allow for changing the browser for the web view. As a result, our only option is to authenticate using the broker app (Microsoft Authenticator app). I followed all the recommended steps outlined in the MSAL documentation for the iOS/iPad operating system (documentation link: https://learn.microsoft.com/en-us/entra/msal/objc/install-and-configure-msal#ios-only-steps), but I am still unable to trigger the broker app for MSAL authentication.
